core/vm/contracts_test.go

```diff
@@ -422,7 +422,7 @@ func TestIsPrecompiledContractEnabled(t *testing.T) {
 			common.BytesToAddress([]byte{3}): &ripemd160hash{},
 			common.BytesToAddress([]byte{4}): &dataCopy{},
 		}
-		if c.config.IsByzantium(c.blockNum) {
+		if c.config.IsEIP198F(c.blockNum) && c.config.IsEIP212F(c.blockNum) && c.config.IsEIP213F(c.blockNum) {
```

Comment:
This is test so it should be fine, but a `mustn'tgrumble` that this can be broken down to three branches for finer control.